src/pages/PersonalProfile/ProfileCard.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import Button from '../../components/UI/Button/Button';
+import styled from 'styled-components';
+import './ProfileCard.css';
+
+const Card = styled.div`
+    width: 20%;
+    height: 90%;
+    text-align: center;
+    margin: 2rem 2.5rem;
+    @media (max-width: 720px) {
+        width: 50%;
```

Comment:
This rule is messing with css. It should be centered.
![width-rule-problem-2](https://user-images.githubusercontent.com/38657595/61593818-20f47000-abed-11e9-86f8-9f1007f34cea.jpg)